src/DynamoCore/Models/NodeModel.cs

```diff
@@ -487,6 +463,8 @@ public bool IsUpdated
                         cachedMirrorData = null;
                     }
                 }
+
+                RaisePropertyChanged("IsUpdated");
```

Comment:
Previously, there was no property change notification for `IsUpdated`. The `Watch3DViewModelBase` knows to ask for render package updates when the `NodeModel.IsUpdated` is changed.